daemons/gptp/common/ieee1588clock.cpp

```diff
@@ -428,14 +420,14 @@ bool IEEE1588Clock::isBetterThan(PTPMessageAnnounce * msg)
 	msg->getGrandmasterIdentity((char *)that1 + 6);
 
 #if 0
-	XPTPD_PRINTF("(Clk)Us: ");
+	GPTP_LOG_DEBUG("(Clk)Us: ");
```

Comment:
Do we need this code removed with if 0? It looks like the GPTP_LOG_DEBUG will only be printing this out when the DEBUG define is set.